Altis_Life.Altis/core/fn_survival.sqf

```diff
@@ -46,9 +46,26 @@ _fnc_water = {
     };
 };
 
+private _fnc_paycheck = {
+    if (alive player) then {
+        private _paycheck = call life_paycheck;
+        if (player distance (getMarkerPos "fed_reserve") < 120 && playerSide isEqualTo west) then {
+            systemChat format [localize "STR_ReceivedPay",[_paycheck + 1500] call life_fnc_numberText];
+            life_atmbank = life_atmbank + _paycheck + 1500;
```

Comment:
Change this `life_atmbank = life_atmbank + _paycheck + 1500;` to `CASH = CASH + _paycheck + 1500;`